db/schema.rb

```diff
@@ -175,6 +160,21 @@
 
   add_index "roles", ["name"], name: "roles_name_idx", unique: true, using: :btree
 
+  create_table "steps", force: :cascade do |t|
+    t.integer  "user_id"
```

Comment:
You look at the `type`, because `steps` is (still) using Single Table Inheritance. Individual steps will either be a `Steps::Approval` or `Steps::Purchase`.